src/content-tags-drawer/data/api.js

```diff
@@ -30,6 +31,7 @@ export const getTaxonomyTagsApiUrl = (taxonomyId, options = {}) => {
 };
 export const getContentTaxonomyTagsApiUrl = (contentId) => new URL(`api/content_tagging/v1/object_tags/${contentId}/`, getApiBaseUrl()).href;
 export const getXBlockContentDataApiURL = (contentId) => new URL(`/xblock/outline/${contentId}`, getApiBaseUrl()).href;
+export const getCourseContentDataApiURL = (contentId) => new URL(`/api/courses/v1/courses/${contentId}/`, getLmsApiBaseUrl()).href;
```

Comment:
Is there a Studio API you can use for this? It's better not to use LMS APIs for the "Course Authoring" frontend if we don't have to.

Reply:
I'm not sure @bradenmacdonald.
I used the same endpoint that brings the course info to the outline:

https://github.com/openedx/frontend-app-course-authoring/blob/6ae9cdac0088117836138e713606c423bec317d5/src/data/api.js#L12-L17

Reply:
OK I guess you can leave it for now then, but I've asked on Slack if there's a better way. I'll let you know if I hear anything better.

Reply:
You can use `/api/contentstore/v1/course_settings/:course_id` which is a CMS API returns the `display_name` . But I guess either way is fine.

Reply:
Nice! Fixed here: 1f14240633ae1f3d7947e947bebbda7b810bc5de